packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboardLanding.tsx

```diff
@@ -48,16 +75,25 @@ export const CloudPulseDashboardLanding = () => {
     <Grid container spacing={2}>
       <Grid item xs={12}>
         <Paper>
-          <GlobalFilters
-            handleAnyFilterChange={onFilterChange}
-            handleDashboardChange={onDashboardChange}
-            handleTimeDurationChange={onTimeDurationChange}
-          />
+          <Box display={'flex'} flexDirection={'column'}>
```

Comment:
```suggestion
          <Box display="flex" flexDirection="column">
```

nit: we don't need object notation here